src/br/analysis/run_features.py

```diff
@@ -0,0 +1,169 @@
+# Free up cache
+import argparse
+import os
+import sys
+
+import pandas as pd
+
+from br.analysis.analysis_utils import (
+    _get_feature_params,
+    _setup_evaluation_params,
+    _setup_gpu,
```

Comment:
FYI the leading underscore convention is that variables named with a leading underscore are internal to the module/class/object they are defined in and not to be used outside of that context. So importing something with a leading underscore breaks the convention. If this is the intended usage, I recommend removing the leading underscores.